### DISCUSSION TITLE
[Question]: How can I manage user access?

### DISCUSSION THREAD

**Discussion by rivindubandara** [danny-avila/LibreChat#3906] [Q&A]
### What is your question?

How can I manage user access?

### More Details

I understand that there is no third party setup like Clerk Auth, so how are users managed. I'm deploying a version for internal company use and would like to see which users are registered, manage passwords, reset password etc.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by kukoboris** [comment]
https://www.librechat.ai/docs/configuration/authentication/OAuth2-OIDC/keycloak

**Comment by spanda1985** [comment]
@kukoboris already provided the link.. 

@rivindubandara - Follow the below to setup local keycloak, before org OIDC we used this in local to test OIDC..

https://www.youtube.com/watch?v=2xOMBdZvsXQ
